src/engine/sparqlExpressions/RegexExpression.cpp

```diff
@@ -285,7 +285,8 @@ PrefixRegexExpression::getPrefilterExpressionForMetadata(
   std::vector<PrefilterExprVariablePair> prefilterVec;
   prefilterVec.emplace_back(
       std::make_unique<prefilterExpressions::PrefixRegexExpression>(
-          prefixRegex_),
+          TripleComponent::Literal::fromStringRepresentation(
+              absl::StrCat("\"", prefixRegex_, "\""))),
```

Comment:
I think there is a better function for this (not dealing manually with the `fromStringRepresenation`, which is an implementation detail